feat(wildberries): Add Get Order integration

dsparill opened the pull request:
# Pull Request: Wildberries Get Order Integration

## Описание изменений

Добавлена интеграция **Wildberries Get Order** для получения информации о конкретном заказе на маркетплейсе Wildberries.

### Что было реализовано:

- **WildberriesGetOrderIntegration** - класс интеграции с поддержкой GET метода
- **Два режима информации** - краткая информация и детальная (с информацией о товарах, адресе доставки, типе оплаты)
- **Полная обработка ошибок** - HTTP статусы 400, 401, 404, 429, 500+, timeout
- **Переменные в параметрах** - поддержка интерполяции переменных для динамических значений
- **Полная валидация** - проверка входных параметров перед API запросом
- **Логирование** - интеграция с BotLogger для отслеживания операций
- **Асинхронная реализация** - использование httpx.AsyncClient для неблокирующих запросов
- **Тесты (9+)** - метаданные, успешные запросы, ошибки, валидация
- **Документация** - техническая документация (GET_ORDER.md) и руководство пользователя (USAGE_GUIDE.md)

## Тип изменений

- [x] Новая функция
- [x] Изменение документации
- [x] Тесты

## Чеклист

- [x] Код соответствует стилю проекта
- [x] Я проверил свой код
- [x] Я добавил комментарии к коду, особенно в сложных местах
- [x] Я обновил документацию (если необходимо)
- [x] Мои изменения не генерируют новых предупреждений
- [x] Я добавил тесты, которые подтверждают мои исправления
- [x] Новые и существующие тесты проходят локально

## Как протестировать

### Локальное тестирование

```bash
# Запустить все тесты Get Order
pytest backend/app/tests/integrations/test_wildberries.py -v

# Запустить конкретный тест
pytest backend/app/tests/integrations/test_wildberries.py::TestWildberriesGetOrderIntegration::test_execute_successful_request -v

# С покрытием кода
pytest backend/app/tests/integrations/test_wildberries.py --cov=app.integrations.wildberries.get_order
```

### Интеграционное тестирование

```python
# Пример использования в DBCV
from app.integrations.registry import registry
from app.auth.credentials_resolver import CredentialsResolver
from app.loggers.bot import BotLogger

# Получить интеграцию
integration = registry.get("wildberries_get_order")

# Конфигурация
config = {
    "order_id": "12345678",
    "detailed": True
}

# Выполнить
result = await integration.execute(
    config=config,
    credentials_resolver=credentials_resolver,
    bot_id=bot_id,
    logger=logger
)

# Результат
print(result["response"]["ok"])  # True/False
print(result["response"]["result"])  # Данные заказа
```

### Примеры запросов

**Краткий запрос:**
```json
{
  "order_id": "12345678",
  "detailed": false
}
```

**Подробный запрос:**
```json
{
  "order_id": "12345678",
  "detailed": true
}
```

**С переменными:**
```json
{
  "order_id": "{{ order_id }}",
  "detailed": "{{ show_details }}"
}
```

## Дополнительная информация

### Файлы, включенные в этот PR

| Файл | Описание | Строк кода |
|------|---------|-----------|
| `backend/app/integrations/wildberries/get_order.py` | Основной файл интеграции | 309 |
| `backend/app/tests/integrations/test_wildberries.py` | Unit тесты (9+ тестов) | 350+ |
| `backend/app/integrations/wildberries/GET_ORDER.md` | Техническая документация | 250+ |
| `backend/app/integrations/wildberries/USAGE_GUIDE.md` | Руководство пользователя | 200+ |

### Обработанные ошибки

| HTTP Статус | Обработка |
|------------|-----------|
| 200 | ✅ Успешный ответ |
| 400 | ❌ Bad Request - ошибка валидации |
| 401 | ❌ Unauthorized - проблема с API ключом |
| 404 | ❌ Not Found - заказ не найден |
| 429 | ❌ Rate Limit - превышен лимит запросов |
| 500+ | ❌ Server Error - ошибка сервера |
| Timeout | ❌ Connection Timeout - истекло время ожидания |

### Совместимость

- ✅ Python 3.9+
- ✅ AsyncIO
- ✅ httpx 0.23+
- ✅ DBCV базовая интеграция (BaseIntegration)

### Известные ограничения

- Максимум одного заказа за раз (для списка заказов используйте `wildberries_get_orders`)
- API ключ должен иметь права на чтение заказов

### Следующие шаги

После мержа этого PR можно реализовать:
1. `wildberries_get_orders` - получение списка заказов с фильтрацией
2. `wildberries_update_stock` - обновление остатков товаров
3. `wildberries_get_analytics` - аналитика по заказам и продажам

dsparill committed:
feat(wildberries): Add Get Order integration

- Implement WildberriesGetOrderIntegration class with GET method
- Support for detailed and brief order information
- Full error handling (400, 401, 404, 429, 500+, timeout)
- Add comprehensive tests (9+ test cases)
- Add technical documentation (GET_ORDER.md)
- Add user guide with examples (USAGE_GUIDE.md)
- Support for variable interpolation in parameters

HadeSxGod1337 commented:
1. Нет видео использования интеграции.
2. В реализации указан провайдер "wildberries", но такого нет. Варианты решений: использовать существующие провайдеры или добавить свой необходимый.
2. Отсутствует регистрация интеграции из-за чего ее не возможно будет создать в интерфейсе, т.к. она банально не придет при апи запросе.

Из-за чего невозможно проверить интеграцию. Прошу исправить недочеты.

dsparill committed:
feat(auth): Add Wildberries API provider for credentials management

- Create WildberriesProvider class for API key management
- Support multiple API key field names (api_token, api_key, token)
- Register provider in AuthService
- Add documentation for provider setup and usage
- Provider implements Bearer token scheme for Wildberries API
- Cache tokens with 1-year TTL (API keys don't expire)
- Full compatibility with existing credential resolver system

dsparill committed:
docs: optimize PR descriptions - remove verbose sections and consolidate information

dsparill committed:
Merge PR #28: Wildberries Get Order Integration

dsparill committed:
Register Wildberries integration import

dsparill committed:
Register Wildberries Get Order integration

dsparill committed:
wildberries: handle 403 Forbidden from API with logging

dsparill committed:
api: add Wildberries provider to providers list for frontend

dsparill committed:
Merge branch 'main' of https://github.com/carbonfay/DBCV

dsparill committed:
wildberries: standardize to api_key and X-API-KEY header

dsparill committed:
schemas: add wildberries to Provider enum

dsparill committed:
feat(integrations): add Wildberries Get Order integration

- Retrieve information about a specific order
- Support for detailed order information
- Uses Wildberries API v2 /api/v2/supplier/orders endpoint
